src/script.js

```diff
@@ -0,0 +1,388 @@
+    name: "Jeane",
+    img: "images/jeane.svg",
+    hair: "brunette",
+    eyes: "green",
+    accessories: ["glasses"],
+    other: []
+  },
+  {
+    name: "Jed",
+    img: "images/jed.svg",
+    hair: "red",
+    eyes: "green",
+    accessories: ["glasses", "hats"],
+    other: ["smoker"]
+  },
+  {
+    name: "Jenni",
+    img: "images/jenni.svg",
+    hair: "white",
+    eyes: "hidden",
+    accessories: ["hats"],
+    other: []
+  },
+  {
+    name: "Jana",
+    img: "images/jana.svg",
+    hair: "black",
+    eyes: "hidden",
+    accessories: ["sunglasses", "jewelry"],
+    other: []
+  },
+  {
+    name: "Jeri",
+    img: "images/jeri.svg",
+    hair: "red",
+    eyes: "green",
+    accessories: ["glasses"],
+    other: []
+  },
+  {
+    name: "Juan",
+    img: "images/juan.svg",
+    hair: "black",
+    eyes: "blue",
+    accessories: [],
+    other: []
+  },
+  {
+    name: "Jocelyn",
+    img: "images/jocelyn.svg",
+    hair: "black",
+    eyes: "brown",
+    accessories: ["glasses", "jewelry"],
+    other: []
+  },
+  {
+    name: "Jordan",
+    img: "images/jordan.svg",
+    hair: "blonde",
+    eyes: "hidden",
+    accessories: ["sunglasses", "hats", "jewelry"],
+    other: []
+  },
+  {
+    name: "Josephine",
+    img: "images/josephine.svg",
+    hair: "white",
+    eyes: "brown",
+    accessories: ["jewelry"],
+    other: []
+  },
+  {
+    name: "Josh",
+    img: "images/josh.svg",
+    hair: "blonde",
+    eyes: "green",
+    accessories: [],
+    other: []
+  },
+  {
+    name: "Jude",
+    img: "images/jude.svg",
+    hair: "black",
+    eyes: "green",
+    accessories: [],
+    other: []
+  },
+  {
+    name: "Julie",
+    img: "images/julie.svg",
+    hair: "black",
+    eyes: "brown",
+    accessories: ["glasses", "hats"],
+    other: []
+  },
+]
+
+
+let secretCharacter;
+let currentQuestion;
+let charactersInPlay;
+let timerInterval;
+
+// Draw the game board
+const generateBoard = () => {
+  gameBoard.innerHTML = ``
+  charactersInPlay.forEach((character) => {
+    board.innerHTML += `
+      <div class="card">
+        <p class="name">${character.name}</p>
+        <img src=${character.img} alt=${character.name}>
+        <div class="guess">
+          <button class="transparent-button" onclick="guess('${character.name}')">Guess <span class="name-block">${character.name}?</span></button>
+        </div>
+      </div>
+    `
+  })
+}
+
+// Randomly select a person from the characters array 
+const setSecretCharacter = () => {
+  secretCharacter = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+}
+
+const startGame = () => {
+  charactersInPlay = characters;
+  generateBoard();
+  setSecretCharacter();
+  guessCount = 0
+  totalGuesses.innerText = "Number of guesses: 0"
+  startTimer()
+}
+
+// setting the currentQuestion object when you select something in the dropdown
+const selectQuestion = () => {
+  questionsOptions.forEach(question => {
+    question.addEventListener("change", () => {
```

Comment:
also a curious question, how come this eventlistener is placed within this function?